pkg/daemon/ceph/client/mgr.go

```diff
@@ -16,6 +18,40 @@ func MgrDisableModule(context *clusterd.Context, clusterName, name string) error
 	return enableModule(context, clusterName, name, false, "disable")
 }
 
+func MgrSetConfig(context *clusterd.Context, clusterName, cephVersionName, key, val string) (bool, error) {
+	var getArgs, setArgs []string
+
+	if cephVersionName == cephv1beta1.Luminous || cephVersionName == "" {
```

Comment:
you shouldn't need to check for the empty string once #2265 is merged, right?

Reply:
IIRC, empty string means `luminous` for compatibility reasons. `initializeDashboard` does the same thing.

Reply:
ok, good idea to do what `initializeDashboard` is doing. That might be an artifact from an earlier implementation of #2187. Now we should be able to rely on it being set, but I'll double check another time.